### PR TITLE
chore: [P4PU-481] updated api endpoint

### PR DESCRIPTION
**Pull request by oleksiybozhykntt**
#### List of Changes
Added endpoint for generating instance of autenticated user for testing purpose

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: openapi/pagopa-arc-be.openapi.yaml [hunk 1]
```diff
@@ -121,6 +121,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorDTO'
+  /auth/testuser:
+    get:
+      tags:
+        - arc auth
+      summary: "Provide access token for the sample user"
+      operationId: getTestUserInfo
```
Comment (and-mora) [inline openapi/pagopa-arc-be.openapi.yaml:129]: not really a 'userInfo', may we found a more suitable name?

File: openapi/pagopa-arc-be.openapi.yaml [hunk 2]
```diff
@@ -121,6 +121,31 @@ paths:
...
+        '200':
+          description: "Token Response"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TokenResponse'
+        '400':
+          description: "Request cannot be processed, required parameters are not present"
+        '401':
+          description: "Unauthorized"
```
Comment (and-mora) [inline openapi/pagopa-arc-be.openapi.yaml:140]: may those responses occur?

**Comment by and-mora** [comment]
not a real `feat` since no value is provided to user.

File: openapi/pagopa-arc-be.openapi.yaml (hunk 1 above)
Comment (oleksiybozhykntt) [reply openapi/pagopa-arc-be.openapi.yaml]: renamed

File: openapi/pagopa-arc-be.openapi.yaml (hunk 2 above)
Comment (oleksiybozhykntt) [reply openapi/pagopa-arc-be.openapi.yaml]: removed

**Review by and-mora — APPROVED** [review]

**Pull request merged**